### PR TITLE
Update NearFutureElectrical for 1.1, primary moved to SpaceDock

### PR DESCRIPTION
**Pull request by lOmicronl**

### CODE REVIEW

**Comment by lOmicronl** [comment]
Not sure why the build failed. JSONLint says the syntax is valid...

File: NetKAN/NearFutureElectrical.netkan [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-    "resources": {
-        "homepage": "http://forum.kerbalspaceprogram.com/threads/52042-0-21"
-    },
+    "spec_version"        : "v1.4",
+    "identifier"          : "NearFutureElectrical",
+    "$kref"               : "#/ckan/spacedock/558",
+    "name"                : "Near Future Electrical",
+    "abstract"            : "Nuclear reactors, capacitors and more batteries that fit right in with stock",
+    "author"              : "Nertea",
+    "license"             : "CC-BY-SA-NC-4.0",
```
Comment (plague006) [inline NetKAN/NearFutureElectrical.netkan]: `"CC-BY-NC-SA-4.0"`

**Comment by lOmicronl** [comment]
Same issue with B9PartSwitch now as in #3717 ...

**Comment by plague006** [comment]
Passes local, holding off on merging until https://github.com/KSP-CKAN/NetKAN/pull/3718 is resolved.

**Comment by plague006** [comment]
Merging this with my thanks.

**Pull request merged**